src/context.py

```diff
@@ -33,15 +102,15 @@ async def on_submit(self, interaction: discord.Interaction):
                 value=field[1] if len(field[1]) < len_max else f"{field[1][:len_max-1]}…",
```

Comment:
This line throws an error, as `reported_user` doesn't have a `len` function.

Reply:
Fixed, accidentally replaced a mention with a user instance

Reply:
It doesn't look like this has been changed?

Reply:
This line was never an issue - the problem was I was referencing `reported_user` instead of `reported_user.mention` several lines below, where the `User` object has no `len` method, but the `str` object obviously does. Feel free to test the changes.

Reply:
UGH nevermind the changes were lost somewhere along the way